plot_equality_data.R

```diff
@@ -55,6 +59,20 @@ if (length(grep(8.5, my_opts[["r"]])) != 0){
 } 
 if (length(variable_rcp) == 0){variable_rcp = c(45,60,85)}
 
+if (my_opts[["s"]] == "all") {
+  ssp_plot = c(1:5) # SSP{1,2,3,4,5
+} else {ssp_plot = c(as.character(my_opts[["s"]]))}
+
+if (is.null(my_opts[["p"]]) || (my_opts[["p"]]== "all") ){
+  variable_timeframe = c("constant")
+} else if (my_opts[["p"]]== "horizon2100"){
+  variable_timeframe = c("horizon2100")
+} else {variable_timeframe = c("constant", "horizon2100")}  
+  
+if (!is.null(my_opts)){
+  test = TRUE
```

Comment:
What? Why would you default to use the test case?

Reply:
test = TRUE if that was given as an option in docopt. If -t is not there it will run the normal case